### DISCUSSION TITLE
Column formatting in a query

### DISCUSSION THREAD

**Discussion by Serhiy-Nazarov** [motor-admin/motor-admin#73] [General]
Hello there!

Is it possible to set column formatting in a query?
I create query with currency column (stored in cents) and I can't show this column correctly in the dashboard.

I have something like `8,150,000` but I need `81 500.00`

Thanks!

**Comment by omohokcoj** [comment]
@Serhiy-Nazarov number formatting depends on your browser - so in your browser locale is en_US - then you'll see `8,150,000` - but users with locale de_DE should see `81 500.00`
